docs/lightning-network/backups.md

```diff
@@ -0,0 +1,125 @@
+
+The backup should be updated every time a channel is opened or closed on the local LN node (including when a _remote_ node opens a channel back to the local node).
+
+The rest of this document details what exactly the SCB-based backup method entails, how to perform backups and restores using it and its limitations.
+
+## Overview of Backup Issues for LN nodes
+
+The next sections provide a high level overview of _why_ backups are needed for Lightning Network nodes and how they work.
+
+For the operational instructions on generating and applying backups see the section on [Operating SCB Backups](#operating-scb-backups).
+
+### Preamble: What data does an LN wallet store?
+
+This can be skipped for those familiar with how LN operates. 
+
+Payments over the Lightning Network happen across _channels_. An LN channel can be understood as an agreement between two entities (nodes) to allow payments up to some pre-defined amount.
+
+While a lightning channel is realized as an on-chain output, spendable by a multisig 2-of-2 signature script, the channel state evolves completely off-chain by having _relative balances_ and _in-flight payments_ (HTLCs) only known to the parties of channel.
+
+This means that while other participants of the network know that a given channel exists (due to the output being stored and confirmed on the blockchain), the data about which portion (how many coins) each of the two nodes own in the channel as well as which payments are currently outstanding is only known to them.
+
+Every sequential update to the channel state is summarized by a corresponding _commitment transaction_: a transaction that is usually kept off-chain (never broadcasted) but that works as a sort of insurance that either party can recover their funds if the other node stops working properly.
+
+This transaction is updated and re-signed every time a new payment changes the relative balances of the channel. It is then stored on a local database, along with some other metadata such as the current state number, the specific keys used to sign this update and information about older, revoked states.
+
+A lightning node must **never** publish a state older than the most recent one agreed by the two parties of a channel. Due to how the LN protocol works, publishing the commitment transaction of an old, revoked state means that party forfeits their funds to the other node, who can then send the _entire_ channel amount to a regular key under their sole control.
+
+Notice that since the commitment data is never published to the blockchain during the regular operation of an LN node, any event that causes loss of data means the node cannot recover their funds if the counterparty disappears.
+
+It also means the node cannot even advance the state of the channel, since it doesn't know what the current state _is_ to begin with. 
+
+### Unsafety of copying the dcrlnd DB
```

Comment:
This heading is not grammatically correct, but I'm not really sure of the best way to re-word it.

Perhaps "Danger of copying the dcrlnd database" ?